addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -162,10 +158,13 @@
                             </a>
                         </t>
                     </t>
+                </td>
+                <td valign="middle">
```

Comment:
That will add an extra TD even if there is no "subtitle" key, which will shift content.
Maybe it's intended?